readme.txt

```diff
@@ -60,6 +60,9 @@ The manual installation method involves downloading our payment plugin and uploa
 Automatic updates should work like a charm; as always though, ensure you backup your site just in case.
 
 == Upgrade Notice ==
+
```

Comment:
Please remove from `== Upgrade Notice ==` and add to` == Changelog ==`

Like this 
`
== Changelog ==
* Added plugin metrics tracker
`